### ISSUE TITLE
Broken Glyphicons

### ISSUE THREAD

**Issue by trentclowater** [dreamingechoes/bootstrap_sb_admin_base_v2#8]
Hi,

In the latest version (0.3.3) glyphicons are broken. This appears to have happened when you updated bootstrap without changing the `src: url(...)` to `font-url(...)` in the Bootstrap `.scss` files. This:

```
@font-face {
  font-family: 'Glyphicons Halflings';
  src: url('../fonts/glyphicons-halflings-regular.eot');
  src: url('../fonts/glyphicons-halflings-regular.eot?#iefix') format("embedded-opentype"), url('../fonts/glyphicons-halflings-regular.woff2') format("woff2"), url('../fonts/glyphicons-halflings-regular.woff') format("woff"), url('../fonts/glyphicons-halflings-regular.ttf') format("truetype"), url('../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format("svg");
}
```
should be changed to this:

```
@font-face {
  font-family: 'Glyphicons Halflings';
  src: font-url("glyphicons-halflings-regular.eot");
  src: font-url("glyphicons-halflings-regular-.eot#iefix") format("embedded-opentype"), font-url("glyphicons-halflings-regular.woff") format("woff"), font-url("glyphicons-halflings-regular.woff2") format("woff2"), font-url("glyphicons-halflings-regular.ttf") format("truetype"), font-url("glyphicons-halflings-regular.svg#glyphicons_halflingsregular") format("svg");		
}
```

**Comment by dreamingechoes** [comment]
Hi trentclowater!

I just released version 0.3.5 to fix this bug, so it should work properly (if not, please tell me in order to fix it). Thank you very much for the issue, as always I appreciate your feedback and work 😊 

Greetings! 😄

**Comment by trentclowater** [comment]
Looks good to me. Good you also caught the similar problem with datatables problem, also.  Thanks for the quick turnaround on getting this published!
